src/style.css

```diff
@@ -0,0 +1,76 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+
+}
+
+.search {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-weight: 300;
+  color: #3d4e61;
+  font-family: Avenir-Book, sans-serif;
+  outline: none;
+  box-shadow: 0 1px 8px white;
+  background: transparent;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+}
+
+.label {
+  box-sizing: border-box;
+  display: block;
+  margin: 20px 8px 0;
+}
+
+.search__big {
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  background-size: 19px 19px;
+  background-position: 25px 24px;
+}
+
+.search__small {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+  background-size: 11px 11px;
+  background-position: 12px 12px;
+}
+
+.search:hover {
+  border: 1px solid rgba(225, 231, 237, 1);
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.search:focus {
+  font-weight: bold;
+  border-radius: 3px;
+  border: 1px solid rgba(255, 231, 237, 1);
+}
+
+.search:default {
+  position: absolute;
+  left: 0%;
+  right: 18.35%;
+  top: 0%;
+  bottom: -15.79%;
```

Comment:
suggestion: do not use such values in the future, we call them ✨ Magic Numbers ✨ – none can understand where they come from, moreover, if there are decimals, hundredths